src/main/java/novamachina/exnihilosequentia/common/compat/waila/CrucibleComponentProvider.java

```diff
@@ -18,6 +18,10 @@ public void appendBody(List<ITextComponent> tooltip, IDataAccessor accessor, IPl
         if (crucibleTile.getFluidAmount() > 0) {
             tooltip.add(new TranslationTextComponent("waila.crucible.fluid", new TranslationTextComponent(crucibleTile.getFluid().defaultFluidState().createLegacyBlock().getBlock().getDescriptionId()), crucibleTile.getFluidAmount()));
         }
-        tooltip.add(new TranslationTextComponent("waila.crucible.heat", crucibleTile.getHeat()));
+        Object heat = crucibleTile.getHeat();
```

Comment:
I'm not a big fan of using the Object class like this. We should actually use the type that we're returning. A tertiary operator could be used to determine what to print.